tasks/pkg-redhat.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Download new RPM key
+  get_url:
+    url: "http://yum.datadoghq.com/DATADOG_RPM_KEY.public.E09422B3"
+    dest: /tmp/DATADOG_RPM_KEY.e09422b3.public
```

Comment:
Correct. We could avoid this by first checking if the key is already imported and only download/import the new key if it's not. I didn't do it because I didn't think it was worth the added complexity, but let me know what you think.